rllib/examples/rock_paper_scissors_multiagent.py

```diff
@@ -52,10 +54,23 @@
     help="Reward at which we stop training.")
 
 
+def env_creator(args):
+    env = rps_v1.env()
+    return env
+
+
+register_env("RockPaperScissors",
+             lambda config: PettingZooEnv(env_creator(config)))
+
+env_for_spaces = PettingZooEnv(env_creator({}))
+obs_space = env_for_spaces.observation_space
+act_space = env_for_spaces.action_space
```

Comment:
Actually, we don't need this anymore. RLlib now automatically infers missing spaces from the workers' env(s)/policies.